synapse/lib/ast.py

```diff
@@ -1849,11 +1852,30 @@ def prepare(self):
         oper = self.kids[1].value()
         self._operfunc = _ExprFuncMap[oper]
 
+    def _coerce(self, parm1, parm2):
```

Comment:
is the expected behavior, that if both values are str, that we return them both?

Reply:
yes, because we can compare strings